crates/omnix-cli/tests/cli.rs

```diff
@@ -19,6 +19,24 @@ fn om_health() -> anyhow::Result<()> {
     Ok(())
 }
 
+/// `om show` runs, and succeeds for a local flake.
+#[test]
+fn om_show_local() -> anyhow::Result<()> {
+    om()?.arg("show").arg(".").assert().success();
+    Ok(())
+}
+
+/// `om show` runs, and succeeds for a remote flake.
+#[test]
+fn om_show_remote() -> anyhow::Result<()> {
+    om()?
+        .arg("show")
+        .arg("github:srid/haskell-multi-nix/c85563721c388629fa9e538a1d97274861bc8321")
+        .assert()
+        .success();
+    Ok(())
+}
+
```

Comment:
Could you add stdout comparison like we do for `om_health` above?

Reply:
Done